tools/docker-network/tests/dockerframework.go

```diff
@@ -349,105 +359,136 @@ func (d *DockerTestFramework) StopIssueCandidacyPayload(nodes ...*Node) {
 		node.IssueCandidacyPayload = false
 	}
 
-	d.DockerComposeUp(true)
+	err := d.DockerComposeUp(true)
+	require.NoError(d.Testing, err)
 }
 
 // CreateTaggedDataBlock creates a block of a tagged data payload.
-func (d *DockerTestFramework) CreateTaggedDataBlock(issuerId iotago.AccountID, tag []byte) *iotago.Block {
-	issuer := d.wallet.Account(issuerId)
+func (d *DockerTestFramework) CreateTaggedDataBlock(issuerID iotago.AccountID, tag []byte) *iotago.Block {
+	issuer := d.wallet.Account(issuerID)
 	ctx := context.TODO()
 	clt := d.wallet.DefaultClient()
 
 	issuerResp, congestionResp := d.PrepareBlockIssuance(ctx, clt, issuer.Address)
 
-	return d.CreateBlock(ctx, &iotago.TaggedData{
+	return d.CreateBlock(&iotago.TaggedData{
 		Tag: tag,
-	}, issuerId, congestionResp, issuerResp)
+	}, issuerID, congestionResp, issuerResp)
+}
+
+func (d *DockerTestFramework) CreateValueBlock(issuerAccountID iotago.AccountID) (*iotago.Block, *iotago.SignedTransaction, *OutputData, iotago.Output) {
+	clt := d.wallet.DefaultClient()
+	ctx := context.Background()
+	currentSlot := clt.LatestAPI().TimeProvider().SlotFromTime(time.Now())
+	apiForSlot := clt.APIForSlot(currentSlot)
+	fundsOutputID := d.RequestFaucetFunds(ctx, iotago.AddressEd25519)
+
+	_, ed25519Addr := d.wallet.Address()
+	input := d.wallet.Output(fundsOutputID)
+	basicOutput := builder.NewBasicOutputBuilder(ed25519Addr, input.Output.BaseTokenAmount()).MustBuild()
+	signedTx, err := builder.NewTransactionBuilder(apiForSlot, d.wallet.AddressSigner(input.AddressIndex)).
+		AddInput(&builder.TxInput{
+			UnlockTarget: input.Address,
+			InputID:      input.ID,
+			Input:        input.Output,
+		}).
+		AddOutput(basicOutput).
+		SetCreationSlot(currentSlot).
+		AllotAllMana(currentSlot, issuerAccountID, 0).
+		Build()
+	require.NoError(d.wallet.Testing, err)
```

Comment:
Let's create a `CreateBasicOutputFromInput` method in wallet and move the tx building logic there, so it's aligned to other methods.